todo/src/reducers/index.js

```diff
@@ -0,0 +1,51 @@
+import {
+  ADD_TODO,
+  TOGGLE_COMPLETED,
+  DELETE_TODO,
+  REMOVE_COMPLETED
+} from "../actions";
+
+const initialState = {
+  todos: [
+    {
+      id: 0,
+      text: "First Todo",
+      completed: false
+    }
+  ]
+};
+
+const reducer = (state = initialState, action) => {
```

Comment:
Good job on getting all of you cases working. That toggle_complete is especially nice.